client/driver/docker.go

```diff
@@ -23,6 +24,28 @@ type DockerDriver struct {
 	fingerprint.StaticFingerprinter
 }
 
+type dockerDriverConfig struct {
```

Comment:
Good Point. I made them private so that it indicates this is not exported outside the package.